src/LayerEditor/ui/data/interface_item.py

```diff
@@ -0,0 +1,20 @@
+from gm_base.geometry_files.format_last import Interface
+
+
+class InterfaceItem:
```

Comment:
Some XYZItem may be simply XYZ from format last. But that should be simplified after whole code is refactored.

Reply:
Possibly. But since it is more of a optimization, it should be done much later. "...Model" and "...Item" are meant to have undoable methods and since this branch lot of these objects got some undoable methods.